### ISSUE TITLE
Publish helm charts

### ISSUE THREAD

**Issue by wonko** [opencv/cvat#7093]
### Actions before raising this issue

- [X] I searched the existing issues and did not find anything similar.
- [X] I read/searched [the docs](https://github.com/cvat-ai/cvat/tree/master#documentation)

### Is your feature request related to a problem? Please describe.

The docs state that the install of CVAT through helm charts should start from the repository. Having published helm charts enables the easier use and distribution of the charts, and using them in automation tooling.

As an example, using the charts with terraform is currently near to impossible, as one would have to either include CVAT as a submodule (and submodules are a hassle to maintain...), pulling the complete CVAT repo each time an automation runs, just to check if the helm chart has an update.

### Describe the solution you'd like

There are many ways to publish helm charts; easiest way might be to use Github pages as the project already uses Github and has Github actions already in place. The marketplace has plenty of tooling which deploys helm charts to GH pages.

**Comment by grzleadams** [comment]
We deploy CVAT with Terraform via the chart, and I just use a null_resource to do the clone:
```
resource "null_resource" "git_clone" {
  triggers = {
    cvat_clone_trigger = var.cvat_ref
    force_clone_trigger = var.force_clone ? timestamp() : 0
  }
  provisioner "local-exec" {
    command = "if [ ! -d ${local.git_clone_path} ]; then mkdir -p ${local.git_clone_path} && git clone https://github.com/opencv/cvat.git ${local.git_clone_path}; fi && cd ${local.git_clone_path} && git checkout ${var.cvat_ref} && helm dependency update helm-chart"
  }
}
```
It's not perfect and can end up in a weird state if you're not careful, but it works fairly well. I agree that a published chart would be preferable, though.

**Comment by grzleadams** [comment]
Agreed; a published chart is better, I was just trying to provide a workaround for anyone who comes along looking for how to use the chart with Terraform (I personally really don't like submodules). 

FWIW the null_resource doesn't pull every time, only when you either set force_clone to true (since timestamp() updates the resource on each run) or change cvat_ref (we pin to specific refs, so it was a natural fit for us). Every other time you run, nothing is changing in state, so it doesn't do the clone. The only problem with that is if you use ephemeral builders, but it feels like using cache in that case makes more sense than re-cloning anyway.

**Comment by azhavoro** [comment]
Resolved #9925
